### ISSUE TITLE
Missing property in Card Holder creation in stripe issuing

### ISSUE THREAD

**Issue by sanalsunnyCq** [stripe/stripe-dotnet#2674]
### Describe the bug

The options for creating cardholder does not contain the latest updates to the individual types in stripe dotnet package

 ie: user_terms_acceptance

Current State
![image](https://user-images.githubusercontent.com/129751996/230131286-b1c1b26f-caa1-4b01-afa3-f72b505cfeda.png)

Missing Items
![image](https://user-images.githubusercontent.com/129751996/230131774-db19a4c7-4e0c-40ea-82de-8d783a998c05.png)

Because of this all card holder are past_due

### To Reproduce

Just check the options of  v1/issuing/cardholders

### Expected behavior

the individual property should have an issuing part

### Code snippets

```csharp
API OUT
  "requirements": {
        "disabled_reason": "under_review",
        "past_due": [
            "individual.card_issuing.user_terms_acceptance.ip",
            "individual.card_issuing.user_terms_acceptance.date"
        ]
    },


MODEL : 

// File generated from our OpenAPI spec
namespace Stripe.Issuing
{
    using Newtonsoft.Json;

    public class CardholderIndividualOptions : INestedOptions
    {
        /// <summary>
        /// The date of birth of this cardholder.
        /// </summary>
        [JsonProperty("dob")]
        public CardholderIndividualDobOptions Dob { get; set; }

        /// <summary>
        /// The first name of this cardholder. This field cannot contain any special characters or
        /// numbers.
        /// </summary>
        [JsonProperty("first_name")]
        public string FirstName { get; set; }

        /// <summary>
        /// The last name of this cardholder. This field cannot contain any special characters or
        /// numbers.
        /// </summary>
        [JsonProperty("last_name")]
        public string LastName { get; set; }

        /// <summary>
        /// Government-issued ID document for this cardholder.
        /// </summary>
        [JsonProperty("verification")]
        public CardholderIndividualVerificationOptions Verification { get; set; }
    }
```

### OS

windows

### .NET version

.NET Core 6

### Library version

41.7.0

### API version

 2022-11-15

**Comment by remi-stripe** [comment]
@sanalsunnyCq You likely are in a beta that gives you access to something that isn't yet public which is why it's not in the library. I recommend reaching out to our support team directly to discuss this and they can help you: https://support.stripe.com/contact

**Comment by remi-stripe** [comment]
ah wait my bad I see the same confusing thing in the public docs so I think I see what you are referring to. I'll flag internally but I'd still recommend talking to our support team directly for this!

**Comment by remi-stripe** [comment]
@sanalsunnyCq Okay sorry I went too fast and trusting your picture. I just went and looked at our real code and I see `CardIssuing` as a real option in `CardholderIndividualOptions` right [here](https://github.com/stripe/stripe-dotnet/blob/master/src/Stripe.net/Services/Issuing/Cardholders/CardholderIndividualOptions.cs#L12). Inside of that it references another class `CardholderIndividualCardIssuingOptions` that you can find [here](https://github.com/stripe/stripe-dotnet/blob/master/src/Stripe.net/Services/Issuing/Cardholders/CardholderIndividualCardIssuingOptions.cs).

This shipped in 41.9.0 per the changelog [here](https://github.com/stripe/stripe-dotnet/blob/master/CHANGELOG.md#4190---2023-03-09) and it seems you are simply a few minor versions behind and didn't update to the latest. If you do, the options will be there as expected!
